ui/src/shared/copy/notifications.ts

```diff
@@ -476,6 +476,12 @@ export const demoDataSucceeded = (
   link,
 })
 
+export const demoDataSwitchedOff = (): Notification => ({
+  ...defaultErrorNotification,
+  message: `Demo data buckets are temporarily unavailable. Please try again later.`,
```

Comment:
is it appropriate to link to https://status.influxdata.com/ even though i know demo data isn't a service that's listed on there?